gefserver/server/access.go

```diff
@@ -131,29 +120,75 @@ func (s *Server) removeTokenHandler(w http.ResponseWriter, r *http.Request) {
 	Response{w}.Ok(jmap("Token", token))
 }
 
-func (s *Server) getLoggedInUser(r *http.Request) (*db.User, error) {
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
I'm using these lines to separate code sections, this makes it more clear to me, but you can remove them if you don't like them 😄